packages/edit-site/src/components/page-pages/index.js

```diff
@@ -150,27 +152,27 @@ export default function PagePages() {
 						</a>
 					);
 				},
-				filters: [ { id: 'author', type: 'enumeration' } ],
-				elements: [
-					{
-						value: '',
-						label: __( 'All' ),
-					},
-					...( authors?.map( ( { id, name } ) => ( {
+				filters: [ 'enumeration' ],
```

Comment:
If a filter is `enumeration` it requires the `elements` prop, right? Why we have them disconnected in the declaration?

Reply:
The rationale is that if a field has `elements`, they'll be used in a few scenarios: filters, when editing the cells, etc.